### PR TITLE
Check if the arguments of the add function are numbers

### PR DESCRIPTION
**Pull request by Puigcerber**
Hey @davyengone! Great workshop yesterday.

I found out in the `add` method that you just check for truthy values and therefore a `string` or `true` won't throw error. In the same manner a "numeric string" would be concatenated and not added. So I think using a function to check if is number and the unary plus operator to convert numeric strings will also help the following attendees with the process of converting the method to ES6.

Cheers, Pablo.

### CODE REVIEW

File: src/arrow-function/index.js [hunk 1]
```diff
@@ -12,10 +12,14 @@ export function add(){
...
+    if(isNumber(arguments[i]))
+      sum += +arguments[i];
     else
       throw new Error('You can only sum numbers');
   }
   return sum;
 }
+
+function isNumber(n) {
+  return !isNaN(parseFloat(n)) && isFinite(n);
```
Comment (davyengone) [inline src/arrow-function/index.js]: Hey Pablo, thanks for your PR. Here are few points to take in account:
- `'2'` should not be converted to `2`
- `isFinite` will parse the value by default so you might want to use `Number.isFinite` (ES6) instead

Comment (davyengone) [inline src/arrow-function/index.js]: Thanks anyway for pointing this out! I guess you just need to remove `parseFloat`and we're good, right?

Comment (Puigcerber) [inline src/arrow-function/index.js]: Yeah, I was wondering if you wanted or not to convert the `'2'` to `2`or not. :)
Thanks for the tip about `isFinite`. ;)

Comment (Puigcerber) [inline src/arrow-function/index.js]: No problem. I'll update the commit straight away!

**Comment by davyengone** [comment]
Great! Thanks man!

**Pull request merged**